src/AppRoot.tsx

```diff
@@ -277,19 +294,60 @@ class AppRoot extends React.Component<RouteComponentProps, State> {
     checkCardInterval = 0 // To indicate setInterval is not running.
   }
 
+  public storeLongValueOnCard = async (longValueBase64: string) => {
+    const formData = new FormData()
+    formData.append('long_value', longValueBase64)
+
+    await fetch('/card/write_long_b64', {
+      method: 'post',
+      body: formData,
+    })
+  }
+
+  public clearLongValueOnCard = async () => {
+    writingVoteToCard = true
+    await this.storeLongValueOnCard('')
+    writingVoteToCard = false
+  }
+
+  public startCardWritePolling = () => {
+    if (cardWriteInterval === 0) {
+      cardWriteInterval = window.setInterval(async () => {
+        if (
+          lastVoteSaveToCard < lastVoteUpdate &&
+          lastVoteUpdate < Date.now() - GLOBALS.CARD_WRITE_DELAY &&
+          !writingVoteToCard
+        ) {
+          lastVoteSaveToCard = Date.now()
+
+          const longValue = btoa(JSON.stringify(this.state.votes))
+
+          writingVoteToCard = true
+          try {
+            await this.storeLongValueOnCard(longValue)
+          } catch (error) {
+            // eslint-disable-next-line no-empty
```

Comment:
I'd replace this with an explanation for why we're ignoring the error.

Reply:
Addressing this: https://github.com/votingworks/bmd/compare/master...catch-long-value-write?expand=1#files_bucket